subprojects/packagefiles/glfw/tests/meson.build

```diff
@@ -0,0 +1,249 @@
+###  GLFW test programs
+#@ Build check
+if opt_tests == false
+  subdir_done()
+endif
+
+#@ Constants
+relpath = '../'
+incdirs = [include_directories(relpath + 'include')
+          ,include_directories(relpath + 'src')
+          ,include_directories(relpath + 'deps')]
+glad_gl = [relpath + 'deps/glad_gl.c']
+glad_vulkan = [relpath + 'deps/glad_vulkan.c']
+getopt = [relpath + 'deps/getopt.c']
```

Comment:
Why are there spaces in the pathnames?

Reply:
I am not certain what you mean. `'../' + 'deps/<filename>' = '../deps/<filename>'` ..correct? They are in unnecessary arrays, however, so I'll fix those up.

Reply:
When I reviewed this on my phone, I could swear I saw stuff like ` getopt = [relpath + ' deps / getopt.c ']`. On the other hand, I could have been insane -- or alternatively, the website was giving my phone a lot of trouble and it may have mis-rendered. idk anymore.